zipkin-collector/kafka/src/main/java/zipkin2/collector/kafka/KafkaCollectorWorker.java

```diff
@@ -103,18 +107,21 @@ public void onPartitionsAssigned(Collection<TopicPartition> partitions) {
           }
         }
       }
-    } catch (InterruptException e) {
-      // Interrupts are normal on shutdown, intentionally swallow
     } catch (RuntimeException | Error e) {
       LOG.warn("Unexpected error in polling loop spans", e);
       throw e;
     } finally {
-      LOG.info("Kafka consumer polling loop stopped.");
-      LOG.info("Closing Kafka consumer...");
-      LOG.info("Kafka consumer closed.");
+      LOG.info("Kafka consumer polling loop stopped. Kafka consumer closed.");
     }
   }
 
+  /**
+   * Stop the polling loop
+   */
+  public void stop() {
```

Comment:
nit this needn't be public, but I'm not concerned about it enough to ask for a revision.